src/core/qgsvectorlayer.h

```diff
@@ -2120,6 +2121,12 @@ class CORE_EXPORT QgsVectorLayer : public QgsMapLayer, public QgsExpressionConte
      */
     void setAllowCommit( bool allowCommit ) SIP_SKIP;
 
+    /**
+     * Returns all the stored expressions for this layer as the manager
+     *
+     * \since QGIS 3.10
+     */
+    QgsStoredExpressionManager *storedFilterExpressions() { return mStoredFilterExpressions; }
```

Comment:
Return reference instead of pointer?

Reply:
It's a QObject and not copyable, hence a pointer looks just fine to me.